app/src/main/java/com/nervousfish/nervousfish/events/PublicKeyReceivedEvent.java

```diff
@@ -15,4 +15,8 @@
     public PublicKeyReceivedEvent(final String key) {
         this.key = key;
     }
+
+    public String getKey() {
```

Comment:
Yeah, according to Stackoverflow it's not needed, the functionality of a getter should be clear from the name. Checkstyle won't give a warning if it's a getter or a setter